### PR TITLE
docs: details on new added modules

### PR DESCRIPTION
**Pull request by PrettyWood**
So we can add https://prql-lang.org/book/reference/stdlib/other-functions/index.html in the changelog as reference

### CODE REVIEW

File: web/book/src/reference/stdlib/other-functions/date.md [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
-manipulate those dates, PRQL has a `date` module with some useful functions
+# Date functions
 
 ### `to_text`
 
-This function allows to convert a date into a text. Since there are many
-possible date representations, `to_text` takes a `format` parameter that
-describes thanks to [specifiers](./format-specifiers.md) how the date or
-timestamp should be structured.
+Converts a date into a text.\
```
Comment (max-sixty) [inline web/book/src/reference/stdlib/other-functions/date.md]: I don't think these are needed in most markdown parsers fwiw

```suggestion
Converts a date into a text.
```

File: web/book/src/reference/stdlib/other-functions/math.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+| ln       | `col`      | Natural logarithm of `col`         |
+| log      | `b` `col`  | `b`-log of `col`                   |
+| log10    | `col`      | 10-log of `col`                    |
+| pi       |            | The constant π                     |
+| pow      | `b` `col`  | Computes `col` to the power `b`    |
+| radians  | `col`      | Converts degrees to radians        |
+| round    | `n` `col`  | Rounds `col` to `n` decimal places |
+| sin      | `col`      | Sin of `col`                       |
+| sqrt     | `col`      | Square root of `col`               |
+| tan      | `col`      | Tangent of `col`                   |
```
Comment (max-sixty) [inline web/book/src/reference/stdlib/other-functions/math.md]: Super, thanks a lot for writing these out!

File: web/book/src/SUMMARY.md [hunk 3]
```diff
@@ -60,6 +58,11 @@
     - [Window](./reference/stdlib/transforms/window.md)
 
   - [Aggregation functions]()
+  - [Other Functions](./reference/stdlib/other-functions/README.md)
+    - [Date functions](./reference/stdlib/other-functions/date.md)
+      - [Date & time format](./reference/stdlib/other-functions/date-time-format-specifiers.md)
+    - [Mathematical functions](./reference/stdlib/other-functions/math.md)
+    - [Text functions](./reference/stdlib/other-functions/string.md)
```
Comment (max-sixty) [inline web/book/src/SUMMARY.md]: No particularly strong view, but I'm not sure we need an "Other" category, we could instead promote those three one level?

File: web/book/src/reference/stdlib/other-functions/date-time-format-specifiers.md
Comment (max-sixty) [inline web/book/src/reference/stdlib/other-functions/date-time-format-specifiers.md:1]: We could promote this to be part of `date.md`; in general I find fewer longer pages are easier to search & navigate, and make the index more information-dense. But no particularly strong view

**Review by max-sixty — APPROVED** [review]
Really good, thanks!

File: web/book/src/reference/stdlib/other-functions/date.md (hunk 1 above)
Comment (PrettyWood) [reply web/book/src/reference/stdlib/other-functions/date.md]: It was to enforce the description on one line and more infos under.

File: web/book/src/SUMMARY.md (hunk 3 above)
Comment (PrettyWood) [reply web/book/src/SUMMARY.md]: I put them at a higher level. LMK if that's better

File: web/book/src/reference/stdlib/other-functions/date-time-format-specifiers.md
Comment (PrettyWood) [reply web/book/src/reference/stdlib/other-functions/date-time-format-specifiers.md:1]: Also done in the same commit.

**Comment by PrettyWood** [comment]
Merging to push the other PR before going to bed. Your remarks have been taken into account. If you want to change them you can do it directly or I'll be happy to do it tomorrow night :)

**Comment by max-sixty** [comment]
It's great!

**Pull request merged**